app/pages/events/event_delete.php

```diff
@@ -4,11 +4,11 @@
 $event_id = get_route_param("event_id");
 $event = em()->find(Event::class, $event_id);
 if (!$event) {
-    echo "the event of id $event_id doesn't exist";
+    force_404("the event of id $event_id doesn't exist");
     return;
```

Comment:
Same thing with `return`. Nice job refactoring this part! 🎉

Reply:
👍👍